images/rootfs.yml.in

```diff
@@ -30,6 +30,10 @@ services:
      image: NEWLOGD_TAG
      cgroupsPath: /eve/services/eve-newlog
      oomScoreAdj: -999
+   - name: edgeview
+     image: EDGEVIEW_TAG
+     cgroupsPath: /eve/services/eve-edgeview
+     oomScoreAdj: -999
```

Comment:
If there is an OOM I'd prefer if edgeview was killed than something else, thus -999 seems odd here.

Reply:
I can change this to '-800', same as the other non-'-999' (eve-rngd) in the EVE

Reply:
Makes sense to change it.